webdriver/tests/sessions/get_timeouts.py

```diff
@@ -0,0 +1,19 @@
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+
+def get_timeouts(session):
+    return session.transport.send("GET", "session/{session_id}/timeouts"
+                                  .format(session_id=session.session_id))
+
+
+# 8.4 Get Timeouts
+
+def test_get_timeouts(session):
+    # 8.4 step 1
+    response = get_timeouts(session)
+
+    assert isinstance(response.body["value"]["script"], int)
+    assert isinstance(response.body["value"]["implicit"], int)
```

Comment:
Would probably also be good for test readability if you checked that `script`, `implicit`, and `pageLoad` were on the `value` object in `response.body` for better readability in case the test should fail.

Reply:
fixed